print/print.env

```diff
@@ -2,3 +2,5 @@ INTERNAL_API_ROOT_URL=http://caddy:3001
 FRONTEND_URL=http://localhost:3000
 SENTRY_PRINT_DSN=
 BROWSER_WS_ENDPOINT=ws://browserless:3000
+PRINT_SERVER=http://print:3003
```

Comment:
Also here, PRINT_URL?

Reply:
Done in https://github.com/ecamp/ecamp3/pull/3036/commits/483415521cf70f767b7e75b9b6d3c4c65df3d741